pynicotine/gtkgui/search.py

```diff
@@ -546,9 +550,15 @@ def on_tooltip(widget, pos_x, pos_y, _keyboard_mode, tooltip):
         return show_file_type_tooltip(widget, pos_x, pos_y, tooltip, 6)
 
     def on_combobox_popup_shown(self, combobox, _param):
+
+        # Refilter
         entry = combobox.get_child()
         entry.emit("activate")
 
+        # Highlight current list item
```

Comment:
Does this actually do anything? The active item is always at the top, and it was already highlighted.

Reply:
Yes, the presets are at the top, so without this the incorrect item was highlighted upon opening the combo popup list.